pandas/_testing.py

```diff
@@ -1755,6 +1755,18 @@ def assert_equal(left, right, **kwargs):
         raise NotImplementedError(type(left))
 
 
+def assert_python_equal(left, right):
+    """
+    Check left and right are equal w.r.t the ``==`` operator.
+
+    Parameters
+    ----------
+    left : object
+    right : object
+    """
+    assert left == right
```

Comment:
ok, I'm guessing this is too much, but I failed to find helper funcs to assert list/tuple equality.
I'll welcome advice on how to write the tests :)

Reply:
yeah don't do this, just do `assert left == right` pytest already handles this

Reply:
ok